### PR TITLE
[l10n] Update translations to Spanish

### PR DESCRIPTION
**Pull request by elima**
Via Weblate, currently translated at 100.0% (651 of 651 strings).

Thanks, @gallegonovato!

### CODE REVIEW

**Comment by gallegonovato** [comment]
Good afternoon @elima

You don't have to thank me for anything. We are here to help :)

The only thing is that a string won't allow me to change it. It is https://hosted.weblate.org/translate/wolvic/-/es/?q=has%3Acheck&offset=4#suggestions with the word Upload , which in Spanish would be: Subir instead of, as translated.

A hug

File: app/src/main/res/values-es-rES/strings.xml [hunk 1]
```diff
@@ -1764,4 +1764,20 @@ the Select` button. When clicked it closes all the previously selected tabs -->
...
     <string name="hamburger_menu_toggle_passthrough">Passthrough</string>
+    <string name="user_feedback_tooltip">Compartir ideas y opiniones</string>
+    <string name="rCN_first_launch_clickable_privacy_text">Política de privacidad</string>
+    <string name="rCN_first_launch_agree">Aceptar</string>
+    <string name="rCN_first_launch_disagree">No estoy de acuerdo</string>
+    <string name="rCN_first_launch_confirmation_body">\\Si seleccionas \"No estoy de acuerdo\", lamentablemente ya no podremos brindarle los servicios.
+\n
+\nPuede leer el Acuerdo de usuario y la Política de privacidad completos.</string>
+    <string name="rCN_first_launch_confirmation_agree">Aceptar y continuar</string>
+    <string name="rCN_first_launch_title">Acuerdo del usuario y la notificación de la política de privacidad</string>
```
Comment (javifernandez) [inline app/src/main/res/values-es-rES/strings.xml:1775]: "Acuerdo del usuario" ? Wouldn't be better "Acuerdo de usuario" ? Just asking.

File: app/src/main/res/values-es-rES/strings.xml [hunk 2]
```diff
@@ -1764,4 +1764,20 @@ the Select` button. When clicked it closes all the previously selected tabs -->
...
+    <string name="rCN_first_launch_clickable_terms_text">Acuerdo del Usuario</string>
+    <string name="rCN_first_launch_body">¡Bienvenidos a “Wolvic”!
+\n
+\nRegistrarte y usar \"Wolvic\" requiere que aceptes el Acuerdo del usuario y la Política de privacidad, o de lo contrario no podemos brindarle los servicios. Lee detenidamente la información y haz clic en \"Aceptar\" para continuar. Si hazes clic en \"No estoy de acuerdo\", no podrás continuar usando \"Wolvic\".
+\n
+\nPuedes leer el Acuerdo de usuario y la Política de privacidad completos.</string>
```
Comment (javifernandez) [inline app/src/main/res/values-es-rES/strings.xml:1781]: In this sentences it's used the term "Acuerdo de usuario". I think we must be consistent on the use of this term.

File: app/src/main/res/values-es-rES/strings.xml [hunk 3]
```diff
@@ -1764,4 +1764,20 @@ the Select` button. When clicked it closes all the previously selected tabs -->
...
+    <string name="rCN_first_launch_disagree">No estoy de acuerdo</string>
+    <string name="rCN_first_launch_confirmation_body">\\Si seleccionas \"No estoy de acuerdo\", lamentablemente ya no podremos brindarle los servicios.
+\n
+\nPuede leer el Acuerdo de usuario y la Política de privacidad completos.</string>
+    <string name="rCN_first_launch_confirmation_agree">Aceptar y continuar</string>
+    <string name="rCN_first_launch_title">Acuerdo del usuario y la notificación de la política de privacidad</string>
+    <string name="rCN_first_launch_clickable_terms_text">Acuerdo del Usuario</string>
+    <string name="rCN_first_launch_body">¡Bienvenidos a “Wolvic”!
+\n
+\nRegistrarte y usar \"Wolvic\" requiere que aceptes el Acuerdo del usuario y la Política de privacidad, o de lo contrario no podemos brindarle los servicios. Lee detenidamente la información y haz clic en \"Aceptar\" para continuar. Si hazes clic en \"No estoy de acuerdo\", no podrás continuar usando \"Wolvic\".
```
Comment (elima) [inline app/src/main/res/values-es-rES/strings.xml:1779]: s/hazes/haces

File: app/src/main/res/values-es-rES/strings.xml (hunk 1 above)
Comment (gallegonovato) [reply app/src/main/res/values-es-rES/strings.xml:1775]: Good afternoon:

You are right I inadvertently increased by one l.

I have already modified it.

Sorry for the mistake.

Best regards

File: app/src/main/res/values-es-rES/strings.xml (hunk 3 above)
Comment (gallegonovato) [reply app/src/main/res/values-es-rES/strings.xml:1779]: Good afternoon:

I really don't understand you here?

But I made some changes in that chain.

Best regards

Comment (elima) [reply app/src/main/res/values-es-rES/strings.xml:1779]: Sorry, I meant that `hazes` should be replaced by `haces`.

**Review by svillar — APPROVED** [review]
Gracias

**Pull request merged**